content/guides/ecommerce-connection/step1-who-does-what.md

```diff
@@ -0,0 +1,99 @@
+:::
+
+## Akeneo PIM and ERP solution...
+
+![Macro overview of the eCommerce connection](../../img/guides/ecommerce-erp-connection-macro.svg)
+
+As seen previously, Akeneo PIM is therefore the source for product information... but what data are we talking about?
+
+### Does Akeneo PIM have all product information needed for eCommerce solution?
+
+The answer is NO because Akeneo PIM only manages **"cold" product information**. It means that Akeneo PIM only owns marketing product information that needs to be checked, organized, enriched. And “cold” product information means that this information does not change every day.
+
+Concerning **“hot product information”** (i.e. product information that changes frequently: prices or stock), this information should be directly transmitted to the eCommerce solution without passing through Akeneo PIM. **“Hot product information”** flow should go directly from the ERP to the eCommerce solution (It’s the ERP's responsibility to deliver this information. Again: a PIM only manages “cold” Marketing product information).
+
+For example, **up-to-date price and stock information** of a product should never be stored in Akeneo PIM and should be transmitted directly from the ERP into the eCommerce solution.
+
+For price information, Akeneo PIM has the capability of storing product price information, but only for a **“reference”** price and not an **“up-to-date”** price. This **“reference” price** is for information purposes only for the teams managing Akeneo PIM (Julia) and should never be mapped to the eCommerce solution price displayed in your store.
+
+:::info
+**Key points:**
+* **PIM** own only **“cold”** product information
+* **ERP** own **“hot”** product information (like price and stock)
+:::
+
+## Akeneo PIM and DAM
+
+![Macro overview of the DAM connection](../../img/guides/dam-pim-connection-macro.svg)
+
+Akeneo PIM can manage product-related media.
+
+As of version 4.0, Akeneo PIM also facilitates the connection with a DAM software thanks to its [new asset management system](/concepts/asset-manager.html).
+
+:::tips
+If you want to know how a DAM connection works don't hesitate to read our [dedicated documentation](../dam-connection/introduction.html).
+:::
+
+There are different 2 ways to manage media with the PIM:
+
+* **Akeneo PIM without a DAM**
+
+Akeneo PIM manages product media internally (with some specific media attributes or its asset manager). Media binaries are then stored directly into the PIM.
+
+* **Akeneo PIM with a DAM**
+
+The DAM solution transfers its data to the PIM (media asset URL and its metadata) and the PIM is then responsible for linking products to media assets.
+
+**Conclusions:**
+
+Instead of binaries stored in the PIM method, the use of media URLs (and therefore the use of a DAM/CDN in addition to a PIM is undoubtedly the best solution to manage product images and allow them to be transferred into the eCommerce solution in a smooth and optimized way as URL information.
```

Comment:
I don't understand this sentence, it's missing this end of the parenthesis.

Reply:
Ok I'll change this sentence.